### ISSUE TITLE
getFollowers do not work!

### ISSUE THREAD

**Issue by serzz1990** [jlobos/instagram-web-api#109]
I get this result. Object data is empty
`{
"count": 4,
"page_info": {
"has_next_page": false,
"end_cursor": null
},
"data": []
}`

**Comment by aymandaher** [comment]
web api has limit 150 requests per hour count*150 requests= max scraped users, another thing to consider is that your account will get web api block eventually if used just for scraping.

**Comment by itxtoledo** [comment]
getFollowers and getFollowings is broken

**Comment by zanotta** [comment]
Yes, having the same problem with both getFollowers and getFollowings. It's not API limit problem, because all other requests works.

**Comment by gabrielbarretoo** [comment]
Some idea to solve the problem?

**Comment by zanotta** [comment]
There were some changes on the Instagram API, I guess we have to wait.

**Comment by miguelopes98** [comment]
Im trying to create a function to get followers given a userInformation (which is the information we get from getUserbyUsername) and i have this exact problem. but if I run api.getFollowers outside that function, I have no problems with that. any idea of why?
